gravitee-apim-cypress/cypress/fixtures/json/updateApi_mockPolicy_example.json

```diff
@@ -0,0 +1,22 @@
+{
+  "paths": {
+    "/": [
+      {
+        "methods": ["GET"],
+        "description": "Description of the Mock Gravitee Policy",
+        "enabled": true,
+        "mock": {
+          "headers": [
+            {
+              "name": "test-value",
+              "value": "value123"
+            }
+          ],
+          "status": "200",
+          "content": "{\n\"message\" : \"This is a mocked response\"\n}"
+        }
+      }
+    ]
+  },
+  "path_mappings": ["/"]
+}
```

Comment:
why using paths ? 
I think it matches the v1 of api definition but we should work with v2 (using plans, floxs, etc)

Reply:
Ok, right. We should prioritise the API v2 and I'll add a test for that. However, since we'll keep v1 for awhile (until 3.20) and the test is now already written I'll also leave that test there for now.